source/setup.Rmd

```diff
@@ -1,282 +1,277 @@
+  (usually Delete, Esc, and/or one of the F# keys). Look for an "Advanced" menu, and under your CPU settings, set the "Virtualization" option
+  to "enabled". Then save the changes and reboot your machine. If you are not familiar with BIOS editing, you may want to find an expert 
+  to help you with this, as editing the BIOS can be dangerous. Detailed instructions for doing this are beyond the scope of this book. 
+More information on installing and running
+Docker on Windows, as well as troubleshooting tips, can be found [in the online
+Docker documentation](https://docs.docker.com/desktop/install/windows-install/).
+
+**Running JupyterLab** Once Docker is running (in either Windows or MacOS), you need to download and run the  
+Docker *image* that we have made available for the worksheets (an *image* is like a "snapshot" of a 
+computer with all the right packages pre-installed). In the search bar, enter `ubcdsci/r-dsci-100`, as this is 
+the name of the image. You will be presented
+with a few options (Figure \@ref(fig:docker-desktop-search)). In the "Tag" drop down menu, click 
+the tag `20230724142640d8f06f` to select the correct version of the image. Then click 
+the "Pull" button to download the image.
+
+```{r docker-desktop-search, echo = FALSE, fig.cap = "The Docker Desktop search window. Make sure to click the Tag drop down menu and find the right version of the image before clicking the Pull button to download it.", fig.retina = 2, out.width="100%"}
+image_read("img/setup/docker-1.png") |> 
+  image_crop("3632x2000")
+```
 
-**MacOS:** To install miniconda on MacOS, you will need to use a different 
-installation method depending on the type of processor chip your computer has.
+Once the image is done downloading, click the "Images" button on the left side of the Docker Desktop window (Figure \@ref(fig:docker-desktop-images)). You
+will see the recently downloaded image listed there under the "Local" tab.
 
-If your Mac computer has an Intel x86 processor chip you can download 
-the [latest Python 64-bit version from here](https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg). 
-After the download has finished, run the installer and accept the default 
-configuration for all pages.
+```{r docker-desktop-images, echo = FALSE, fig.cap = "The Docker Desktop images tab.", fig.retina = 2, out.width="100%"}
+image_read("img/setup/docker-2.png") |> 
+  image_crop("3632x2000")
+```
 
-If your Mac computer has an Apple M1 processor chip you can download 
-the [latest Python 64-bit version from here](https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh).
-After the download has finished, you need to run the downloaded script in the terminal using a command
-like:
+To start up a *container* using that image, click the play button beside the image. This will open the run configuration menu (Figure \@ref(fig:docker-desktop-runconfig)).
```

Comment:
Note: on a Mac I had to also click on the down arrow/carrot to see the configuration menu (on Mac it is called "Optional settings"

Reply:
<img width="537" alt="Screenshot 2023-08-28 at 12 13 06 PM" src="https://github.com/UBC-DSCI/introduction-to-datascience/assets/6865016/d0144322-303a-4f5c-9e14-c5620be93aee">

<img width="532" alt="Screenshot 2023-08-28 at 12 13 14 PM" src="https://github.com/UBC-DSCI/introduction-to-datascience/assets/6865016/55cd981e-1e36-47d8-9bf1-ca970b57dd10">

Reply:
added a remark to open the drop down if necessary